Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 appVersion: v0.1.0
 description: This chart deploys the GlueOps Platform
 name: glueops-platform
-version: 0.5.0
+version: 0.6.0
```

Comment:
@fernandoataoldotcom make sure you update the admiral repo to use this new version as well

Reply:
good call, will do